internal/storage/db.go

```diff
@@ -24,7 +27,16 @@ SET
 WHERE
 	issuance_week_id = $1;`
 
-func (s *DBStorage) AssignTokens(ctx context.Context, issuanceWeek int, totalTokens *big.Int) error {
-	_, err := s.DBS.DBS().Writer.ExecContext(ctx, assignTokensQuery, issuanceWeek, totalTokens.String())
+func (s *DBStorage) AssignTokens(ctx context.Context, issuanceWeek, firstAutomatedWeek int) error {
```

Comment:
is `firstAutomatedWeek` ever going to change?

Reply:
Nope!